docs/csharp/programming-guide/classes-and-structs/instance-constructors.md

```diff
@@ -13,46 +13,54 @@ Instance constructors are used to create and initialize any instance member vari
   
  The following example shows an instance constructor:  
   
- [!code-csharp[csProgGuideObjects#5](~/samples/snippets/csharp/VS_Snippets_VBCSharp/csProgGuideObjects/CS/Objects.cs#5)]  
+ [!code-csharp[CoordsWithParameterlessConstructorOnly#1](snippets/instance-constructors/coords/Program.cs#1)]
   
 > [!NOTE]
 > For clarity, this class contains public fields. The use of public fields is not a recommended programming practice because it allows any method anywhere in a program unrestricted and unverified access to an object's inner workings. Data members should generally be private, and should be accessed only through class methods and properties.  
   
  This instance constructor is called whenever an object based on the `Coords` class is created. A constructor like this one, which takes no arguments, is called a *parameterless constructor*. However, it is often useful to provide additional constructors. For example, we can add a constructor to the `Coords` class that allows us to specify the initial values for the data members:  
   
- [!code-csharp[csProgGuideObjects#76](~/samples/snippets/csharp/VS_Snippets_VBCSharp/csProgGuideObjects/CS/Objects.cs#76)]  
+ [!code-csharp[TwoArgumentConstructor#2](snippets/instance-constructors/coords/Program.cs#2)]
   
  This allows `Coords` objects to be created with default or specific initial values, like this:  
   
- [!code-csharp[csProgGuideObjects#77](~/samples/snippets/csharp/VS_Snippets_VBCSharp/csProgGuideObjects/CS/Objects.cs#77)]  
+ [!code-csharp[InstantiatingCoords#3](snippets/instance-constructors/coords/Program.cs#3)]
   
  If a class does not have a constructor, a parameterless constructor is automatically generated and default values are used to initialize the object fields. For example, an [int](../../language-reference/builtin-types/integral-numeric-types.md) is initialized to 0. For information about the type default values, see [Default values of C# types](../../language-reference/builtin-types/default-values.md). Therefore, because the `Coords` class parameterless constructor initializes all data members to zero, it can be removed altogether without changing how the class works. A complete example using multiple constructors is provided in Example 1 later in this topic, and an example of an automatically generated constructor is provided in Example 2.  
   
  Instance constructors can also be used to call the instance constructors of base classes. The class constructor can invoke the constructor of the base class through the initializer, as follows:  
   
- [!code-csharp[csProgGuideObjects#78](~/samples/snippets/csharp/VS_Snippets_VBCSharp/csProgGuideObjects/CS/Objects.cs#78)]  
+```csharp
+class Circle : Shape
+{
+    public Circle(double radius)
+        : base(radius, 0)
+    {
+    }
+}
+```
```

Comment:
This contains a compile error as the class doesn't implement the abstract methods from base.
This seems intentional because later in the article, the complete example have the complete class in the complete sample. So I moved this part of code to the article itself.